application-services/custom/camera-management/README.md

```diff
@@ -1,19 +1,117 @@
+
+2. Add User to group:
+   ```bash
+   sudo usermod -aG docker $USER
+   ```
+
+3. Refresh the group:
+   ```bash
+   newgrp docker 
+   ```
+
+4. To verify the Docker installation, run `hello-world`:
+
+   ```bash
+   docker run hello-world
+   ```
+   A **Hello from Docker!** greeting indicates successful installation.
+
+   ```bash
+   Unable to find image 'hello-world:latest' locally
+   latest: Pulling from library/hello-world
+   2db29710123e: Pull complete 
+   Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+   Status: Downloaded newer image for hello-world:latest
+
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ...
+   ```
+
+### Install Docker Compose
+Install Docker Compose from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/linux/#install-using-the-repository) site.
+
+### Install Tools
+Install the build, media streaming, and parsing tools:
+
+```bash
+sudo apt install build-essential
+```
+
 ## Steps for running this example:
-We expect you'll be running in a relatively modern Linux environment
-with `docker`, `docker-compose`, and `make` installed.
 
-1. Get the [EdgeX Core Services][edgex-core-services] and Edgex device ONVIF service running by referring to docs from
-   [Edgex Onvif device service][device-onvif-camera].
-2. Get [Edge Video Analytics Microservice][evam] running for inference.
+### 1. Start the EdgeX Core Services and Device Services.
+
+1. Clone `edgex-compose` from github.com.
+   ```shell 
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```  
+
+2. Navigate to the EdgeX `compose-compose` directory:
+
+   ```bash
+   cd edgex-compose
+   ```
+
+3. Checkout latest branch.
+   ```shell
+   git checkout levski
+   ```
+
+Note: The `levski` branch is the latest stable branch at the time of this update. 
+
+4. Navigate to the EdgeX `edgex-compose/compose-builder` directory:
+
+   ```bash
+   cd compose-builder/
+   ```
+
+5. Update the `add-device-usb-camera.yml` file:
+
+   a. Add the rtsp server hostname environmental variable to the `device-usb-camera` service.
+   ```yml
+   services:
+      device-usb-camera:
+             environment:
+               DRIVER_RTSPSERVERHOSTNAME: `your-local-ip-address`
+   ```
+      where `your-local-ip-address` is the ip address of the machine running the rtsp server.
+
+   b. Under the `ports` section, find the one for port 8554 and change the host_ip from 127.0.0.1 to either 0.0.0.0 or the ip address you put in the previous step.
+6. (Optional) If onvif cameras are being used, please refer to the [Edgex Onvif device service](https://github.com/edgexfoundry/device-onvif-camera/blob/main/doc/guides/CustomStartupGuide.md) documentation to run the onvif device service with camera credentials.
```

Comment:
Replace this step with the old `3. Configure Camera Credentials`, noting it is only for onvif cameras.

Also, we should add this note from the configuration.toml file, into that old section  here as well:

```
    # NOTE: currently this solution is limited to supporting only 1 username/password combination
    #       for ALL cameras. In the future when then device-onvif-camera service is able to provide
    #       us with pre-authenticated uris, this can be removed.
```

Reply:
was `uris` suppose to be `devices` on the last line?

Reply:
> was `uris` suppose to be `devices` on the last line?

@brianointel no, but you can change it to `stream uris` to make it more clear